public/js/admin.controller.js

```diff
@@ -0,0 +1,22 @@
+import angular from 'angular';
+
+export default angular.module('helloGov')
+.controller('adminController', function($scope, $http, $location, constants) {
+    'ngInject';
+    $scope.loginDetails = {};
+    $scope.signupDetails = {};
+
+    $scope.createLink = function() {
+        $http.get(`${constants.API_ROOT}/createLink?email=${$scope.newUserDetails.email}`)
+        .then(function(data) {
+            $scope.showSignupLink = true;
+            $scope.signupLink = $location.host() + '/signup/' + data.data;
+        })
+        .catch(function(data) {
+            console.log('Error: ' + data);
+        });
+    };
+
+    $scope.showSignupLink = false;
+})
+.name;
```

Comment:
With angular 1 you have to export the `module()`'s name (a string, in this case 'adminController'), because of the way angular's dependency injection works. With other more modern frameworks you'd simply export the module itself, maybe a `function` or `class` or `var`.